Userland/Utilities/pkg/InstalledPort.cpp

```diff
@@ -38,29 +38,22 @@ ErrorOr<HashMap<String, InstalledPort>> InstalledPort::read_ports_database()
             // FIXME: Skip over invalid entries instead?
             return Error::from_string_view("Database entry too short"sv);
         }
-        auto string_type = parts[0];
-        auto name = TRY(String::from_utf8(parts[1]));
+        auto port_name = TRY(String::from_utf8(parts[1]));
 
-        if (auto maybe_type = type_from_string(string_type); maybe_type.has_value()) {
+        if (auto maybe_type = type_from_string(parts[0]); maybe_type.has_value()) {
```

Comment:
previously this function gave a name to parts[0] of `string_type`. Why did you remove the useful name for the indicies of the parts[] list?

Reply:
No idea, maybe because "why init new variable if we can use parts[0]"?